.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: python
+python:
+    - 3.6
+before_install:
+    - sudo rm -f /etc/boto.cfg
+install:
+    - pip install boto3
```

Comment:
Can we do a `pip install -r requirements.txt` here? That way we can be assured our code is running tests with the same version of libraries that our code is running in production with.